doc/modules/index.rst

```diff
@@ -8,7 +8,7 @@ This is the class and function reference of nilearn. Please refer to
 the :ref:`user guide <user_guide>` for more information and usage examples.
 
 .. toctree::
-   :hidden:
+   :maxdepth: 1
```

Comment:
allows to have a listing of the main sections: https://output.circle-artifacts.com/output/job/355bc487-2984-47fa-86cf-365d85904bac/artifacts/0/dev/modules/index.html

That otherwise do not display on mobile:

Current:
![image](https://github.com/user-attachments/assets/edb08acd-381a-4c40-81f6-b9b944adda1a)



In this PR:
![image](https://github.com/user-attachments/assets/0ea5f7d2-b5ac-4183-bc8e-fdeebee3c431)